scripts/run-spark.py

```diff
@@ -66,6 +68,18 @@ def get_args(*args, **kwargs):
         type=int,
         help="number of cores per executor"
     )
+    parser.add_argument(
+        "--num_runs",
+        default=1,
+        type=int,
+        help="number of times to run sort"
+    )
+    parser.add_argument(
+        "--mapred_split_size",
+        default= 128 * (10 ** 6),
```

Comment:
Could we leave the default to -1, and in TeraSort check if it is -1, then don't set the split size. Also make a note that -1 means default, which is 128MB according to the sources we found?